### ISSUE TITLE
Rescan Bluetooth device

### ISSUE THREAD

**Issue by Gummibaer** [thorsten-gehrig/alexa-remote-control#28]
its possible run Speak command to rescan Bluetooth device
"looking for new devices"
("suche neue Geräte" in german)

**Comment by Apollon77** [comment]
No. Not possible via speak. It is only possible using other aPI
